deploy/crds/cincinnati.openshift.io_cincinnatis_crd.yaml

```diff
@@ -43,6 +43,12 @@ spec:
               description: GitHubRepo is the repository to use on GitHub for retrieving
                 additional graph data.
               type: string
+            graphDataImage:
+              description: GraphDataImage is an init container image that contains the
```

Comment:
Did you add this description manually or via comment? I think we should rely on the CRD being generated from controller-tools (so we don't accidentally lose something important in the future).

Reply:
It was manually. Thanks for the reminder. I've updated _types.go and regenerated the CRD using operator-sdk.